### PR TITLE
Network accounts

### PR DESCRIPTION
**Pull request by kyungmin**

### CODE REVIEW

File: src/balanced.js [hunk 1]
```diff
@@ -31,6 +31,9 @@ var capabilities = {
     })(),
 };
 
+var root_url = 'https://api.balancedpayments.com',
+marketplace_href = null;
+dialog = null;
```
Comment (mjallday) [inline src/balanced.js]: these need the `var` keyword in front of them

File: src/balanced.js [hunk 2]
```diff
@@ -31,6 +31,9 @@ var capabilities = {
     })(),
 };
 
+var root_url = 'https://api.balancedpayments.com',
```
Comment (mjallday) [inline src/balanced.js]: this param should be on the balanced object so it can be configured

File: src/balanced.js (hunk 1 above)
Comment (mjallday) [inline src/balanced.js]: by setting dialog as a global you will lose the reference if you call this method a second time.

```
// first call
dialog = window.open(xxx)
// second call
dialog = window.open(aaa)
// how do we reference the first instance of the dialog?
```

File: src/balanced.js (hunk 2 above)
Comment (matthewfl) [inline src/balanced.js]: This can be configured using the init call

File: src/balanced.js [hunk 3]
```diff
@@ -384,7 +387,65 @@ var ba = {
...
+    },
+    configure: function () {
+	/*jsonp(make_url('/jsonp/'+marketplace_href+'/configs', {}), function (json) {
+	    ea.network = json;
+	});*/
+    }
+};
+
+addEvent(window, 'message', function (event) {
+    if(event.origin !== "https://js.balancedpayments.com") {
```
Comment (mjallday) [inline src/balanced.js:431]: should be set on the balanced object so we can configure it for testing

File: src/balanced.js (hunk 1 above)
Comment (matthewfl) [inline src/balanced.js]: it looks like this was done b/c the message listener is global, so how would you tell from two different tokens coming in at the same time.  I suppose we could prevent multiple calls to the create method?

File: test/unit/external_accounts.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        });
+    },
+
+    teardown: function () {
+    }
+});
+
+asyncTest('tokenizeExternalAccount', function (assert) {
+    balanced.externalAccount.create('test', function (result) {
+	start();
```
Comment (matthewfl) [inline test/unit/external_accounts.js]: can you finish this test?

Comment (kyungmin) [inline test/unit/external_accounts.js]: I'm currently working on this, but will get back to this after balanced.js gets merged in.

Comment (matthewfl) [inline test/unit/external_accounts.js]: ? 
The callback.html is on the js.bp.com domain, why do you need this merged first?

**Comment by matthewfl** [comment]
can you change the node version in the .travis.yml file to 10 to see if that fixes travis

File: src/balanced.js [hunk 5]
```diff
@@ -454,13 +514,29 @@ if(typeof JSON !== 'object') {
...
 }
 
-global.balanced = {
+var balanced = global.balanced = {
+    root_url: 'https://api.balancedpayments.com',
     card: cc,
     bankAccount: ba,
+    externalAccount: ea,
     emailAddress: em,
+    jsonp: jsonp,
```
Comment (matthewfl) [inline src/balanced.js:523]: I would much rather you not do this and just make api calls when testing stuff.

Comment (mjallday) [inline src/balanced.js:523]: ok. +1 on @matthewfl's idea.

**Pull request merged**